config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml

```diff
@@ -55,6 +55,9 @@ spec:
          # Default nic config template for a EDPM compute node
          # These vars are edpm_network_config role vars
          edpm_network_config_hide_sensitive_logs: false
+         ctlplane_dns_nameservers:
```

Comment:
You don't have  to add it here, should be populated with IPAM[1]

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/pkg/deployment/inventory.go#L113

Reply:
Agree with Rabi and Brendan in that I'm not sure this is needed. @Jaganathancse The commit message should explain what is being fixed, and describe the issue, and how it's fixed. Can you add some more detail about why this PR might be needed?